tests/Unit/Lock/lock_host_pointer.cpp

```diff
@@ -0,0 +1,24 @@
+// XFAIL: Linux
+// RUN: %hc %s -o %t.out %t.out
+
+#include <hc.hpp>
+
+int main()
+{
+    hc::accelerator acc;
+    const auto& all = hc::accelerator::get_all();
+    if(all.size() == 0){
+        return -1;
+    }
+    std::vector<hsa_agent_t> agentVec;
```

Comment:
please consider not using HSA data structures directly in hcc unit tests.

if you must use them. please refer to:
https://github.com/RadeonOpenCompute/hcc/blob/master/tests/Unit/HC/create_marker.cpp
for an example. you'll basically need to change lit macro to make your test build.